### PR TITLE
[W5][M11-1]Lee Chong Wei Justin

### PR DESCRIPTION
**Pull request by ghost**
Added SortCommand

### CODE REVIEW

**Comment by yingrong1996** [comment]
Nice feature. The implementation seems to be quite efficient

File: docs/UserGuide.adoc [hunk 1]
```diff
@@ -156,6 +156,11 @@ Views all details of the 1st person in the results of the `find` command.
 Clears all entries from the address book. +
 Format: `clear`
 
+== Sorts all persons : `sort`
```
Comment (okkhoy) [inline docs/UserGuide.adoc:159]: Good job updating the documentation!

File: src/seedu/addressbook/data/person/ComparePerson.java [hunk 2]
```diff
@@ -0,0 +1,14 @@
+package seedu.addressbook.data.person;
+
+import java.util.Comparator;
+
+/**
+ * Compares person by name
```
Comment (okkhoy) [inline src/seedu/addressbook/data/person/ComparePerson.java:6]: good job on following the standards when you write the header comments. 👍

**Review by okkhoy — COMMENTED** [review]
Good job on the code updates.
- Remember to update the tests next time! 
- Try to follow the convention for github commit messages `Add SortCommand` vs `Added SortCommand`

**Comment by okkhoy** [comment]
@E0176106 
Some comments added. Please take note and close the pull request.

**Pull request closed without merging**